sdk/communication/AzureCommunication/Source/Authentication/CommunicationAccessToken.swift

```diff
@@ -27,8 +27,8 @@
 import Foundation
 
 @objcMembers public class CommunicationAccessToken: NSObject {
-    let token: String
-    let expiresOn: Date
+    public let token: String
```

Comment:
This change has been merged right? I am curious why it shows up as a diff in this PR?

Reply:
Yes I merged in that change. This should be resolved when I update the fork